functions/async/java/pubsub/README.md

```diff
@@ -0,0 +1,39 @@
+# Autoscaling service based on queue depth
+
+## Prerequisites
+
+You can refer to the [Installation Guide](https://openfunction.dev/docs/getting-started/installation/) to setup OpenFunction.
+
+Follow [this guide](../../../../Prerequisites.md#kafka) to install a Kafka server named `kafka-server` and a Topic named `sample-topic`.
```

Comment:
The prerequisites link can be changed to https://openfunction.dev/docs/getting-started/quickstarts/prerequisites/